pkgs/applications/virtualization/rust-hypervisor-firmware/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, makeRustPlatform
+, hostPlatform
+, targetPlatform
+, lld
+}:
+
+let
+  arch = targetPlatform.qemuArch;
+
+  target = ./. + "/${arch}-unknown-none.json";
+
+  cross = import ../../../.. {
+    system = hostPlatform.system;
+    crossSystem = lib.systems.examples."${arch}-embedded" // {
+      rustc.config = "${arch}-unknown-none";
+      rustc.platform = builtins.fromJSON (
+        builtins.readFile target
+      );
+    };
+  };
+
+  rustPlatform = cross.makeRustPlatform {
+    inherit (cross.rustPlatform.rust) rustc cargo;
+  };
+
```

Comment:
is this just `inherit (cross) rustPlatform`?

Reply:
Indeed! That didn't even cause a rebuild.